frontend/src/client.js

```diff
@@ -65,7 +100,10 @@ export const currentUserVar = makeVar({
 })
 
 const httpLink = createHttpLink({
-  uri: '/graphql',
+  uri:
+    process.env.NODE_ENV === 'production'
+      ? 'https://tracker.alpha.canada.ca/graphql'
+      : '/graphql',
```

Comment:
Tracker is going to be deployed at https://tracker.canada.ca and is currently running at https://staging.tracker.canada.ca and https://tracker.ops.cyber.gc.ca as well as our GCP environment at https://tracker.alpha.canada.ca.
I think leaving this at `/graphql` is probably a good idea.